src/helpers/index.test.ts

```diff
@@ -105,23 +105,23 @@ describe('isPublished', () => {
 
   test('returns false if article is published in the future', () => {
     const unpublishedArticle: PublishableItemType = {
-      publishedDate: DateTime.now().plus({ weeks: 2 }).toISO(),
+      publishedDate: DateTime.now().plus({ weeks: 2 }).toISO()!,
```

Comment:
Since this is in a test and just setting up test data, I'm good with just adding the `!`.